.travis.yml

```diff
@@ -15,7 +15,7 @@ before_script:
   - export PYTHONPATH=$PYTHONPATH:$(pwd)
 
 script:
-    - py.test --cov=soepy
+    - py.test --cov=soepy --doctest-modules
```

Comment:
can we set up the (already improved) pytest.ini so that there is no additional command line options needed here?

Reply:
Do you also want to get rid of the --cov=soepy command or do you only refer to the --doctest-modules one?